frame/elections-phragmen/src/lib.rs

```diff
@@ -1110,7 +1110,13 @@ impl<T: Config> Pallet<T> {
 							})
 							.collect::<Vec<_>>()
 							.try_into()
-							.expect("Cannot accept more than DesiredMembers."),
+							.unwrap_or_else(|_| {
```

Comment:
There exists `defensive_unwrap_or_default` which does that.

Reply:
Well yes, that is what I did, I updated the code in the previous two commits.